File: MeetingWithJS/FINAL_JS_REVESION/JAVA/Basicsss/Prime.java
public class Prime {
    public static boolean checkPrime(int n){
        if(n <= 1){
            return false;
        }
        
        for(int i = 2; i <= n/2; i++){
            if(n % i == 0){
                return false;
            }
        }
        return true;
        
    }
    public static void main(String... args){
        int n = 0;
        boolean flag = checkPrime(n);
        
        if(flag){
            System.out.println("Number is prime: ");
        }else{
            System.out.println("Number is not prime: ");
        }
    }
}

File: MeetingWithJS/FINAL_JS_REVESION/JS/Basic22222/Prime.js
function checkPrime(n){
    if(n <= 1){
        return false;
    }

    for(let i = 2; i <= parseInt(n/2); i++){
        if(n % i == 0){
            return false;
        }
    }

    return true;
}

(function (){
  let flag =  checkPrime(11);
  if(flag){
    console.log('Number is prime: ');
  }else{
    console.log("Number is not prime: ");
  }
})();